server/handler.go

```diff
@@ -190,3 +190,19 @@ func (h Handler) KillTask(conf *config.Config) http.HandlerFunc {
 		writeJSON(respStatus, body, w)
 	}
 }
+
+func (h Handler) DeleteTask(conf *config.Config) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		id := vars["taskId"]
+		logrus.WithField("task_id", id).Debug("Deleting task")
+		err := h.database.DeleteTask(id)
+		respStatus := http.StatusAccepted
```

Comment:
If the task doesn't exist, this won't respond with 404, it'll either be accepted or internal server error. Both of these feel a bit weird to me.